### ISSUE TITLE
[Bug] Cannot sideload on iPhone 15 Pro (invalid bundle)

### ISSUE THREAD

**Issue by Rexios80** [arichorn/uYouPlusExtra#125]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

- iOS/iPadOS version: 17.0.2
- Device model: iPhone 15 Pro Max
- Sideload tool (AltStore, Sideloadly, TrollStore,...): iOS App Signer
- The specific version of uYouPlus (**latest or newest is NOT a version number!**): v18.35.4-3.0.1

### Describe the bug

```console
Error installing  '/Users/rexios/Downloads/uYouPlusExtra.ipa', ERROR: Error Domain=com.apple.dt.CoreDeviceError Code=3002 "Failed to install the app on the device." UserInfo={NSUnderlyingError=0x60000a3c2eb0 {Error Domain=com.apple.dt.CoreDeviceError Code=3000 "The item at uYouPlusExtra.ipa is not a valid bundle." UserInfo={NSURL=file:///Users/rexios/Downloads/uYouPlusExtra.ipa, NSLocalizedDescription=The item at uYouPlusExtra.ipa is not a valid bundle., NSLocalizedFailureReason=Failed to read the bundle.}}, NSURL=file:///Users/rexios/Downloads/uYouPlusExtra.ipa, NSLocalizedDescription=Failed to install the app on the device.}
Domain: com.apple.dt.DVTCoreDevice
Code: -1
User Info: {
    DVTErrorCreationDateKey = "2023-09-22 15:08:07 +0000";
}
--


System Information

macOS Version 13.6 (Build 22G120)
Xcode 15.0 (22265) (Build 15A240d)
Timestamp: 2023-09-22T11:08:07-04:00
```

### Steps to reproduce the issue

1. Repackage with iOS App Signer
2. Attempt to install with Xcode

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

### Additional context

Installing the same IPA on an older iPhone works

**Comment by Rexios80** [comment]
Should we create an issue in iSponso Block? Is it open source?

**Comment by nalin29** [comment]
It works on iPhone 15 using it right now

**Comment by Rexios80** [comment]
Im pretty sure this issue is specific to the iPhone 15 Pro because of the new CPU

**Comment by jxjftw** [comment]
Installed on 15 Pro Max using Altstore, no issues, works fine.

**Comment by hakufu** [comment]
Installed on 15 Pro with Sideloady. Also no issues. Paid dev account though.

**Comment by senorYME** [comment]
i use altstore , to update ios 17 you either disable sip or use version 1.7 of altstore. from what altstore devs mention on patreon to make altstore ios17 to work you need latest windows altserver or mac os sonoma

**Comment by jxjftw** [comment]
Installed using Mac OS 13.5, didn’t need 14.

**Comment by Rexios80** [comment]
Okay so what is AltStore/Sideloadly doing that the app signer isn't? I used this exact method to install on my 14 pro last week.

**Comment by robpneu** [comment]
I also use iOS App Signer w/ a paid dev account on my MacBook Pro running macOS 13.6 and it is failing to install for me with the same message.

I think it's related to how it was signed than the iOS device itself as other apps I had signed and worked just fine on my iPhone 12 are also failing. I wonder if AltStore/Sideloadly are using a newer version of macOS.

**Comment by Rexios80** [comment]
[This issue](https://github.com/DanTheMan827/ios-app-signer/issues/239) suggests installing the IPA with Apple Configurator which actually works. This seems to be a problem with Xcode installing IPAs on iOS 17 devices and not an issue with the iOS App Signer.

**Comment by Rexios80** [comment]
I updated the older iPhone to iOS 17 and can confirm it no longer works for that device either

**Comment by robpneu** [comment]
I now agree that this is an xcode on iOS/iPadOS 17 issue. I tested 4 different apps on 3 different devices:
- iPhone 15 Pro: iOS 17.0.2
- iPhone 12: iOS 17.0.1
- iPad Pro (11-inch) (3rd generation): iPadOS 17.0.1

I did a fresh re-sign using iOS App Signer with my paid Apple Developer account on known working .ipa's and then tried installing resulting (signed) .ipa's, first using xcode's Devices and Simulators and then using Apple Configurator. Every one failed using xcode with the same error message (`ERROR: Error Domain=com.apple.dt.CoreDeviceError Code=3002 "Failed to install the app on the device."`) and worked flawlessly (and faster in my opinion) with Apple Configurator. Also of note, I'd originally installed all of these apps on my iPhone 12 and iPad when they were on iOS/iPadOS 16 without issue and they still worked after I'd updated them to iOS/iPadOS 17 last week. Only when I removed them and tried to reinstall them with xcode caused any issue. I'm not going to downgrade either to iOS/iPadOS 16 to exhaustively test this but I'm pretty confident this is an issue with xcode and iOS/iPadOS 17.

I forgot Apple Configurator even existed and never thought to use it in conjunction with iOS App Signer. I found using Apple Configurator even easier than the xcode Devices and Simulators window method anyways so I'm probably just going to switch to it permanently.
